doc/faq.md

```diff
@@ -0,0 +1,24 @@
+# Dragonfly Frequently Asked Questions
+
+- [What is the license model of Dragonfly? Is it an open source?](#what-is-the-license-model-of-dragonfly-is-it-an-open-source)
+- [Can I use dragonfly in production?](#can-i-use-dragonfly-in-production)
+- [Dragonfly provides vertical scale, but we can achieve similar throughput with X nodes in a Redis cluster.](#dragonfly-provides-vertical-scale-but-we-can-achieve-similar-throughput-with-x-nodes-in-a-redis-cluster.)
+- [I get it, but why not change Redis open source?](#i_get_it_but_why_not_change_redis_open_source)
+- [What if I have some commands that are missing?](#what_if_i_have_some_commands_that_are_missing)
+
+
+## What is the license model of Dragonfly? Is it an open source?
+Dragonfly is released under [BSL 1.1](../LICENSE.md) (Business Source License). The BSL license is more permissive than AGPL-like licenses in some elements. In general terms, it means that dragonfly's code is free to use and free to change as long as you do not provide paying support or managed services for an in-memory datastore.
+We followed the trend of other technological companies like Elastic, Redis, MongoDB, Cockroach labs, Redpanda Data to protect our rights to provide service and support for the software we are building. 
+
+## Can I use dragonfly in production?
+License wise you are free to use dragonfly in your production as long as you do not provide dragonfly as a managed service. From a code maturity point of view, Dragonfly's code is tested with unit testing and also against the Redis testing TCL scripts. However as with any new software there are use cases that are hard to test and predict. We advise you to run your own particular use case on dragonfly for a few days before considering production usage. 
```

Comment:
`as long as you do not sell services directly related to dragonfly or in-memory datastores` .

Reply:
Do we have covrage stats?

Reply:
No, and there is definitely a place for improvement there in terms of coverage. 
But the thing is that we can do high-precision testing at all levels.